File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/home/HomeController.java
package com.isystk.sample.web.admin.controller.html.home;

import static com.isystk.sample.common.AdminUrl.HOME;

import com.isystk.sample.web.base.controller.html.AbstractHtmlController;
import org.slf4j.Logger;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping(HOME)
public class HomeController extends AbstractHtmlController {

  private static final Logger log = org.slf4j.LoggerFactory.getLogger(HomeController.class);

  @Override
  public String getFunctionName() {
    return "A_HOME";
  }

  @GetMapping
  public String index(Model model) {
    return "modules/home/index";
  }

}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/home/HomeController.kt
package com.isystk.sample.web.admin.controller.html.home

import com.isystk.sample.common.AdminUrl
import com.isystk.sample.web.base.controller.html.AbstractHtmlController
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping(AdminUrl.HOME)
class HomeController : AbstractHtmlController() {
    override fun getFunctionName(): String {
        return "A_HOME"
    }

    @GetMapping
    fun index(model: Model?): String {
        return "modules/home/index"
    }

}